crates/bevy_pbr/src/meshlet/from_mesh.rs

```diff
@@ -72,6 +71,7 @@ impl MeshletMesh {
         let vertex_buffer = mesh.create_packed_vertex_buffer_data();
         let vertex_stride = mesh.get_vertex_size() as usize;
         let vertices = VertexDataAdapter::new(&vertex_buffer, vertex_stride, 0).unwrap();
+        let vertex_normals = bytemuck::cast_slice(&vertex_buffer[12..16]);
```

Comment:
Is this simply a way of getting a pointer to the first vertex's normal, so it can be passed to `meshopt`?
And it can safely assume it's the 12-bytes in as the attributes have been validated by `validate_input_mesh `?
https://github.com/bevyengine/bevy/blob/ca34e974837009d8a807e91d24366fe72c83f9bb/crates/bevy_pbr/src/meshlet/from_mesh.rs#L230-L239

Thanks in advance, just checking my knowledge and piecing this together in my head.

Reply:
Yep that's correct. Not the cleanest piece of code, but oh well.